.github/workflows/check.yml

```diff
@@ -18,7 +18,7 @@ jobs:
         uses: actions-rs/toolchain@v1
         with:
           profile: minimal
-          toolchain: 1.76.0
+          toolchain: stable
```

Comment:
any particular reason for this? I hardcoded the toolchain versions as I was cautious of specifying release channels because of potential breaking changes.

Though this made me curious and I looked around at our dependencies across the stack, and I see no consistency across the board. Some use stable, some hardcode, some have both stable and hardcoded toolchains in the same CI file.

That being said, I think I'm cool with setting it to stable because it will require attention only when it breaks, as opposed to now updating it when we need to? Curious what you think.

Reply:
So basically I'm using functionality in the tests which isn't compatible with those previous versions. See:
https://github.com/timewave-computer/covenants/actions/runs/9812118600/job/27095575968
Since the optimizer uses a specific version that is compatible for wasmd anyways, it won't matter for the contracts. I think having stable everywhere is better so we keep it consistent.